source/jquery.flot.js

```diff
@@ -984,6 +987,25 @@ Licensed under the MIT license.
             executeHooks(hooks.bindEvents, [eventHolder]);
         }
 
+        function addEventHandler(event, handler, eventHolder, priority) {
+            var key = eventHolder + event;
+            var eventlist = eventManager[key];
```

Comment:
So, we have <code>event**H**older</code>, <code>event**M**anager</code>, and <code>event**D**ata</code> (below), but <code>event**l**ist</code> here?  Why does this variable get a special naming convention?

Reply:
Inertia.